src/api/methods/getTransactionReceipt.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2019-present, Leap DAO (leapdao.org)
+ *
+ * This source code is licensed under the Mozilla Public License Version 2.0
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+const { Tx } = require('leap-core');
+const getPrevTx = require('./utils/getPrevTx');
+
+module.exports = async (db, hash) => {
+  const txDoc = await db.getTransaction(hash);
+  if (!txDoc) return null;
+
+  const { txData, blockHash, height, txPos } = txDoc;
+
+  const tx = Tx.fromJSON(txData);
+
+  const prevTx = await getPrevTx(db, tx);
+
+  return {
+    transactionHash: hash,
+    transactionIndex: txPos,
+    blockHash,
+    blockNumber: `0x${height.toString(16)}`,
+    from: tx.from(prevTx),
+    to: tx.to(),
+    cumulativeGasUsed: '0x0',
+    gasUsed: '0x0',
+    contractAddress: null,
+    logs: [],
+    logsBloom: '0x',
+    status: 1,
```

Comment:
Should be a hex string (according web3 docs and [geth code](https://github.com/ethereum/go-ethereum/blob/8812c4d3f9f57edff6706608b93a1c5567e57d9f/internal/ethapi/api.go#L1282))